crypto/ec/ec_key.c

```diff
@@ -263,7 +263,7 @@ static int ecdsa_keygen_knownanswer_test(EC_KEY *eckey, BN_CTX *ctx,
 
     st = OSSL_SELF_TEST_new(cb, cbarg);
     if (st == NULL)
-        return 0;
+        goto err_point;
```

Comment:
Instead of adding a new label I would move `EC_POINT_new(eckey->group)` after OSSL_SELF_TEST_onbegin()